edx_sga/sga.py

```diff
@@ -249,6 +249,7 @@ def upload_assignment(self, request, suffix=''):
         user = self.get_real_user()
         require(user)
         upload = request.params['assignment']
+        sha1 = get_sha1(upload.file)
```

Comment:
What's the significance of this change?

Reply:
some how file.tell() is effecting sha1 calculating, after the sha1 algo sends same hash. which is replacing files